lib/src/conflicts.rs

```diff
@@ -473,7 +473,12 @@ fn parse_conflict_hunk(input: &[u8]) -> Merge<BString> {
         }
     }
 
-    Merge::from_removes_adds(removes, adds)
+    if adds.len() == removes.len() + 1 {
+        Merge::from_removes_adds(removes, adds)
+    } else {
+        // Doesn't look like a conflict
```

Comment:
I'd say something like "We cannot parse these conflict markers" or "Doesn't look like a valid conflict"

Reply:
Done (here and above)